lib/turn.rb

```diff
@@ -0,0 +1,80 @@
+class Turn
+  attr_reader :player1, :player2, :spoils_of_war
+
+  def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @spoils_of_war = []
+  end
+
+  def type
```

Comment:
There's a bit of repetition in this method that could benefit from some helper methods to reduce the amount we are repeating ourselves, as well as make this easier to read. For example on lines 11 and 13 you have this same code:
`player1.deck.rank_of_card_at(0) == player2.deck.rank_of_card_at(0)`

You could write a method like this to use instead:
```ruby
  def same_rank_at_zero?
    @player1.deck.rank_of_card_at(0) == @player2.deck.rank_of_card_at(0)
  end
```
And then use that method wherever you check this logic in the code.